src/components/basic/display/SubtextAmount/index.tsx

```diff
@@ -30,7 +31,7 @@ export interface Props extends WrapperProps {
   amount: string | React.ReactElement;
 }
 
-export const SubtextAmount = (props: Props): JSX.Element => {
+const _SubtextAmount = (props: Props): JSX.Element => {
```

Comment:
I don't know why it's not a linting rule, I thought it was, but not a fan of "dangling underscores", maybe rename to "UnsafeSubtextAmount" to indicate it's usage without `memo` is unsafe as it will cause looping errors.

Reply:
Sure, will rename them.
Just a note, the loop was caused by a mistake in a callback dependency list.
The `memo`s are purely for performance/prevent unnecessary re-renders when parent re-renders without actually changing any props.
For this reason I was considering even exposing 2 versions:
1. Regular named component: `Subtext`
2. Memoed component: `MemoSubtext` / `SubtextMemo`

Or something like that. Although, not sure there will ever be a use case for the non memoized component.

What do you think?